app/src/main/java/com/r5k/contacerveja/ui/base/MVPView.kt

```diff
@@ -1,6 +1,4 @@
 package com.r5k.contacerveja.ui.base
 
 interface MVPView{
-    fun showProgress()
-    fun hideProgress()
 }
```

Comment:
This interface is implemented/extended by BaseActivity, BaseFragment, DrinkMVPView, and MainMVPView.
Since MVPView is now an Empty interface, shouldn't we remove it? 
Or should we keep it in order to use in the future to implement new features?

Reply:
This is the base interface in a MVP architecture. It’s empty because we basically have just one activity. When we add more features and activities this should be used.